test/cql-pytest/README.md

```diff
@@ -38,7 +39,14 @@ to control which tests to run:
   For `pytest` to support the `--count` option, you need to install a
   pytest extension: `pip install pytest-repeat`
 
-Additional useful pytest options, especially useful for debugging tests:
+#### How to run from PyCharm IDE
```

Comment:
What's exactly the motivation to run from PyCharm instead of just typing "pytest"?
It looks like it only makes things more complicated?
Also, I assume this will require you to run Scylla manually before starting the test, right?

Reply:
Done